apps/docs/content/3-components/2-library/5-button.mdx

```diff
@@ -2,13 +2,249 @@
 title: Button
 description: Button HTML Component
 draft: false
-status: in-development
+status: stable
```

Comment:
Shouldn't this be in-development? @pregno

Reply:
I think that is the status of the documentation.
Below there is the status of the library and is set to `alpha`